lib/vehicle_factory.rb

```diff
@@ -0,0 +1,13 @@
+class VehicleFactory
+  attr_reader :vehicle
+  def initialize
+    @vehicles = []
+  end
+
+  def create_vehicles(vehicles)
+    vehicles.map do |vehicle|
+      car = Vehicle.new(vin: vehicle[:vin_1_10], year: vehicle[:model_year], make: vehicle[:make], model: vehicle[:model], engine: :ev)
```

Comment:
`Vehicle` initializes with a single argument - a hash. You are passing each attribute in individually. It should probably look a little more like:
```ruby
      car = Vehicle.new(
        {
          vin: vehicle[:vin_1_10],
          year: vehicle[:model_year],
          make: vehicle[:make],
          model: vehicle[:model],
          engine: :ev
        }
      )
```